### PR TITLE
Improve a transcript the-changelog-365.md

### PR DESCRIPTION
**Pull request by janisz**
https://github.com/thechangelog/transcripts/issues/23

### CODE REVIEW

File: podcast/the-changelog-365.md [hunk 1]
```diff
@@ -168,8 +168,6 @@ We could start to think about having lots of message passing between components,
 
 **Chris Bell:** Or restarts. If we've got supervisors, we've got restart strategies. So we can say "That bit of the UI will boot back up into its previous state, or some known good state."
 
-**Desmond Bowe:** \[unintelligible 00:32:52.25\]
-
```
Comment (jerodsanto) [inline podcast/the-changelog-365.md]: Why the delete? Did Desmond just not say anything substantive here?

Comment (janisz) [reply podcast/the-changelog-365.md]: For me it sounds like he accidentally turned on his mic.

Comment (pnevares) [reply podcast/the-changelog-365.md]: Looks like someone else figured out it as "That’s true again." for this transcript.

**Comment by pnevares** [comment]
Another contributor was able to resolve these unintelligibles in a PR, thanks for the try @janisz!

**Pull request closed without merging**